src/main/java/lotto/domain/LottoWinnerType.java

```diff
@@ -4,46 +4,66 @@
 import java.util.Map;
 
 public enum LottoWinnerType {
-    FIRST_PLACE(6, 2000000000),
-    SECOND_PLACE(5, 1500000),
-    THIRD_PLACE(4, 50000),
-    FOURTH_PLACE(3, 5000),
-    NON_WIN(-1, 0);
+    FIRST_PLACE(6, false, 2000000000),
```

Comment:
```suggestion
    FIRST_PLACE(6, false, 2_000_000_000),
```
위와같이 가독성을 향상 시킬수 있습니다.

Reply:
아... 저는 언더스코어를 넣으면 문자형이 되는 줄 알았는데 아니네요. 가독성이 훨씬 보기 좋습니다!🙂